draft/0000-name-main-command-django.rst

```diff
@@ -0,0 +1,184 @@
+
+Backwards Compatibility
+=======================
+
+The existing ``django-admin`` command will remain indefinitely
+as an alias of ``django`` command.
+There are no plans to deprecate or remove the ``django-admin`` alias.
+
+Motivation
+==========
+
+Django is how many people first learn Python,
+so the choices that Django makes have an outsized impact
+on the intuition they have of how things work in Python.
+This makes it more important that Django
+follow Python's simple and clean style,
+and match the conventions of the broader ecosystem.
+
+Naming the main command ``django``
+can reduce new developer confusion and make it easier to remember
+by following the most typical patterns in both the Python ecosystem
+and in the broader software development world.
+Some Python examples include ``pip``, ``pytest``, and ``black``,
+and some broader examples include ``ember``, ``rails``, and ``vite``.
+
+The broad acceptance of this pattern has been reinforced
+by tools like ``uv tool run`` (aka ``uvx``) and ``pipx``,
+where commands that are the same as the package name
+get special privileges and a simpler syntax.
+For example ``uvx pytest`` automatically downloads and runs
+the ``pytest`` command from the ``pytest`` package.
+
+This pattern has been further reinforced
+by the common pattern of recommending to use, for example,
+``python -m pip`` to ensure that
+you're using the version of the module
+that is associated with your intended Python interpreter.
+The correspondance between the command name and the package name
+allows for a more intuitive mapping to the alternative style.
+
+Some commenters have described this alternative as
+intuitive, fun, aesthetic, and modern.
+These subjective benefits
+are not sufficient motivation to make the change alone,
+and are likely to be largely based on the intuitions built
+from the motivations above,
+but they reinforce values that we desire in the project.
+
+Drawbacks
+=========
+
+All changes have some drawbacks, this is no exception.
+It is important to consider them,
+and to make sure that they are mitigated
+or are outweighed by the benefits.
+
+* There is work involved with updating and maintaining
+  the implementation and documentation,
+  that could potentially go to other tasks.
+* There are many tutorials and blog posts
+  that reference the existing ``django-admin`` command,
+  and while these will continue to work,
+  the authors may reasonably think it wisest to update them.
+  This is work that will be done by the community,
+  so we should be cautious with adding this burden.
+* Having multiple ways to spell the same thing can be confusing.
+  This concern is especially relevant because of the amount
+  of external resources that reference the existing command name.
+* The ``django`` command can be seen as a ambiguity conflict
+  between the name of the command and the name of the python package,
+  or the name of the project itself.
```

Comment:
Possibly, but the rebuttal is that most frameworks with a CLI already do this. Django is the exception here and it would be more intuitive to go with the pack.

I think it would be good to address these perceived drawbacks in addition to just listing them, as most have a good answer.